README.md

```diff
@@ -88,6 +88,48 @@ The easiest way to run release please is as a GitHub action:
   convention, [release-please](https://github.com/googleapis/release-please)
   will start creating Release PRs for you.
 
+#### Automating publication to npm
+
+With a few additions, the Release Please action can be made to publish to
+npm when a Release PR is merged:
+
+```yaml
+on:
+  push:
+    branches:
+      - master
+name: release-please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: bcoe/release-please-action@v1.2.1
+        id: release
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          release-type: node
+          package-name: test-release-please
```

Comment:
Generally, can't you get this out of `package.json`?

Reply:
Yeah, we do now detect this from `package.json` in the case of Node.js repos, so we could drop this line.